extensions/linux/dictionary.json

```diff
@@ -17,6 +17,36 @@
       "caption": "Effective Group ID",
       "description": "The effective group under which this process is running.",
       "type": "integer_t"
+    },
+    "linux_kernel_info": {
+      "caption": "Linux Kernel Information",
+      "description": "Details about the kernel running on the Linux endpoint.",
+      "type": "linux_kernel_info"
+    },
+    "kernel_release": {
+        "caption": "Kernel Release",
+        "description": "The kernel release of the operating system. Determined from the <code>uname -r</code> command output on Unix-based systems.",
+        "type": "string_t"
+    },
+    "kernel_version": {
+        "caption": "Kernel Version",
+        "description": "The kernel version of the operating system. Given the full version is w.x.y-z, the kernel version is w.",
```

Comment:
The terminology might get confusing here and even though you've already been really precise, there might be room for even more clarity...

When I run `uname -r` on my Fedora box is reports "6.8.8-200.fc39.x86_64". When I run it at my WSL2 prompt it reports "5.15.153.1-microsoft-standard-WSL2". You refer to this entire string above as the kernel release but I think a lot of people will tend to think of this string as being the full version. But it's clear from the way you describe the full version in this comment that you are referring only to the 4-tuple numeric part at the start.

So I'm just wondering if instead of using "w.x.y-z" as the sample string you instead use the kind of string that will be returned by `uname -r`. Might somebody think, for example, that the patch in my WSL2 example is 1-microsoft-stanrdard-WSL2 instead of just 1?

Reply:
Sure I can update the sample string to make it more clear. Also @antchan2 and I were debating the version fields yesterday. It seems like for Linux there are no general terms for which part of the version field we are representing. So, he suggested maybe instead of `kernel_version`, `major_revision` `minor_revision` and `patch_number` we use `version_first`, `version_second`, `version_third`, and `version_fourth`, respectively.

Reply:
[Change kernel version names](https://github.com/rebeccasenger/ocsf-schema/pull/1/commits/5b04fc7eff1d813c1dd756b6909b4d39cf9556fc)